plugins/woocommerce/includes/class-wc-deprecated-action-hooks.php

```diff
@@ -117,6 +117,9 @@ public function handle_deprecated_hook( $new_hook, $old_hook, $new_callback_args
 	 * @return mixed
 	 */
 	protected function trigger_hook( $old_hook, $new_callback_args ) {
+		$order_id = null;
+		$item_id = null;
+		$item = null;
```

Comment:
I think the proper fix here is just to remove the below case statements as they will never actually fire an action:

```
			case 'woocommerce_order_update_coupon':
			case 'woocommerce_order_update_shipping':
			case 'woocommerce_order_update_fee':
			case 'woocommerce_order_update_tax':
```

Reply:
Done 318fe824600808a8a6bf6b97a406d86d4bb65d7c